.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,35 +1,25 @@
-## PR Checklist
-Please check if your PR fulfills the following requirements:
-
-- [ ] Tests for the changes have been added (for bug fixes / features)
-- [ ] Docs have been added / updated (for bug fixes / features)
+## PR Description 
+<!-- Expected Commit Message Description (imported automatically by GitHub) -->
+<!-- Must conform to [conventional commits guidelines](https://github.com/edgexfoundry/app-functions-sdk-go/blob/master/.github/CONTRIBUTING.md) -->
 
-### Related Docs PR now required (if applicable) 
-Related Docs PR:
+Closes/Fixes #IssueNumber 
 
-If n/a for Docs PR, state why it is not applicable:
+<!-- Detailed description of change if no related issue -->
 
 <!-- If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/master/.github/CONTRIBUTING.md -->
 
-## What is the current behavior?
-<!-- Please describe the current behavior and link to a relevant issue. -->
-
-Issue Number:
-
-## What is the new behavior?
-<!-- Please describe the new behavior. -->
-
-## Does this PR introduce a breaking change?
-<!-- If this PR contains a breaking change, if so please describe the impact and migration path for existing applications below. -->
-
-- [ ] Yes
-- [ ] No
-
-## Are there any new imports or modules? If so, what are they used for and why?
+## PR Checklist
+Please check if your PR fulfills the following requirements:
 
-- [ ] Yes
-- [ ] No
+- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
+- [ ] I am not introducing a new dependency (add notes below if you are)
+- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
+- [ ] I have fully integration tested this the new feature or bug fix (if not, why?)
```

Comment:
What constitutes integration testing?  Does that mean firing up the framework locally and making sure it doesn't burn down and fixes the problem?

Reply:
Yes, if appropriate for the change.

Reply:
My suggestion would be to drop the word "integration" from this.

Reply:
Personally I like it, or some other description that conveys that one has actually booted the framework with the fix in place.